packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -143,9 +126,23 @@ export async function getProjectIdAsync(
 
   Log.warn('EAS project not configured.');
 
+  const getAccountNameForEASProjectSync = (exp: ExpoConfig, user: Actor): string => {
+    if (exp.owner) {
```

Comment:
Not sure if it should be part of this PR, but there needs to be a better way to select an account. To do it now, you need to define it in app.json, run a command that adds projected, and then remove it.